contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -1232,6 +1232,46 @@ $$
 STRICT
 LANGUAGE plpgsql IMMUTABLE;
 
+CREATE OR REPLACE FUNCTION sys.daydiff_between_dates(IN d1 INTEGER, IN m1 INTEGER, IN y1 INTEGER, IN d2 INTEGER, IN m2 INTEGER, IN y2 INTEGER) RETURNS INTEGER AS $$
```

Comment:
This looks like a helper function, should we add it in sys_function_helpers.sql script?

Reply:
done